### ISSUE TITLE
Access improvements

### ISSUE THREAD

**Issue by andyError** [layersoflondon/application#398]
- Tab index favours map record pins over form elements
- Elements should have sufficient colour contrast (93)
- Images must have alt text (6)
- Form elements must have labels (1)
- Zooming and scaling must not be disabled
- Page must contain one main landmark (1)
- Content must be contained in a landmark region (1)

**Comment by edtjones** [comment]
We need to use ARIA Live regions as well https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
